Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Merchant.hs

```diff
@@ -38,13 +38,18 @@ findByShortId shortId_ = do
     where_ $ merchant ^. MerchantShortId ==. val (getShortId shortId_)
     return merchant
 
-findByExoPhone :: Transactionable m => Text -> Text -> m (Maybe Merchant)
-findByExoPhone countryCode exoPhone = do
+findByExoPhone :: Transactionable m => Text -> m (Maybe Merchant)
+findByExoPhone exoPhone = do
```

Comment:
Is this exophone unique? Because if two merchants will have the same exoPhone, this query will throw an error